tests/integration/test_schema_registry_auth.py

```diff
@@ -198,8 +218,7 @@ async def test_sr_auth_forwarding(registry_async_auth_pair: List[str]) -> None:
         while True:
             assert counter < max_tries, "Compat update not propagated"
             resp = requests.get(f"{primary_url}/config", auth=auth)
-            if not resp.ok:
-                continue
+            assert resp.ok
```

Comment:
Is this change intentional? It will fail now instead of retrying.

Reply:
As we have a retry alreadt, maybe no need to continue, but then we have this check above `assert counter < max_tries, "Compat update not propagated"`, not sure if it's used for the old retry style.

Reply:
No, good catch.

Reply:
Done.